pkg/embed/gzip.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2021 The Jaeger Authors.
```

Comment:
@Ashmita152 was this implementation copied from somewhere? If so, we need to add attribution.